### DISCUSSION TITLE
Is there a way to edit the Feed URL after creating an entry?

### DISCUSSION THREAD

**Discussion by mdkberry** [Athou/commafeed#1067] [Q&A] [answered]
I am trying to fiddle with using rsshub to get Twitter feeds working in commafeed, and it requires a lot of changing of entries in the feed URL to tweak it to look right. Currently I do this by recreating the entry each time and seeing how it looks.

Is there a way to edit the Feed URL and then refresh that entry instead? The Feed URL seems uneditable.

**Comment by Athou** [comment, marked as answer]
It's not possible at the moment to just change the feed URL because multiple users may be subscribed to the same feed and it would change the URL for all users subscribed to it.
Adding an option to change the URL would be possible but it would essentially be a shortcut for "unsubscribe to old url" then "subscribe to new url".
